specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -1898,6 +1911,36 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoiceSections/{invoiceSectionName}/elevate": {
+      "post": {
+        "description": "Elevates the caller's access to match their billing profile access.",
+        "operationId": "InvoiceSection_ElevateToBillingProfile",
```

Comment:
Here InvoiceSection needs to be plural so it doesn't collide with type `InvoiceSections`, plus it will group all operations with the prefix `InvoiceSections_` in the same interface.